definitions/npm/react-css-modules_v3.x.x/flow_v0.53.x-/test_react-css-modules-v3.js

```diff
@@ -14,12 +14,28 @@ class ExampleModule extends Component<{
   }
 }
 
+type Props = {};
+
+type State = {};
+
+const ExampleModule2 = (props: Props) => {
+  return (<div>hoge</div>)
+}
+
+const ExampleModule3 = (props: Props, state: State) => {
+  return (<div>hoge</div>)
+}
+
 const ExampleCSSModules = CSSModules(ExampleModule, styles);
 const ExampleCSSModules2 = CSSModules(ExampleModule, styles, { allowMultiple: true });
+const ExampleCSSModules3 = CSSModules(ExampleModule2, styles);
+const ExampleCSSModules4 = CSSModules(ExampleModule2, styles, { allowMultiple: true });
+const ExampleCSSModules5 = CSSModules(ExampleModule3, styles);
+const ExampleCSSModules6 = CSSModules(ExampleModule3, styles, { allowMultiple: true });
```

Comment:
Want to do some extra credit and use `describe` and `it` blocks to make all our testing lives easier? :-)

Reply:
> some extra credit
hmm... For example, what kind of test is it? 🤔

Reply:
So, rather than doing `ExampleCSSModules1`, `ExampleCSSModules2`, etc., do...

```javascript
describe('blah', () => {
  it('should do blah', () => {
    ...
  })
})
```

It'll help us all in understand what is already being tested, since it's not always obvious. :)

Reply:
OKay, thanks. I will rewrite some code.

Reply:
@AndrewSouthpaw 
PTAL 🙇 https://github.com/flowtype/flow-typed/pull/2019/commits/c4e16a8d94d3a8070c92d0409578d8aa104eedb9